FTB/Signatures/tests.py

```diff
@@ -874,528 +874,782 @@ def runTest(self):
-        self.assertEqual(crashInfo.backtrace[8], "js::jit::InvokeFunction")
-        self.assertEqual(crashInfo.backtrace[9], "??")
-        self.assertEqual(crashInfo.backtrace[10], "EnterIon")
-        self.assertEqual(crashInfo.backtrace[11], "js::jit::IonCannon")
-        self.assertEqual(crashInfo.backtrace[12], "Interpret")
-        self.assertEqual(crashInfo.backtrace[13], "js::RunScript")
-        self.assertEqual(crashInfo.backtrace[14], "js::ExecuteKernel")
-        self.assertEqual(crashInfo.backtrace[15], "js::Execute")
-        self.assertEqual(crashInfo.backtrace[16], "Evaluate")
-        self.assertEqual(crashInfo.backtrace[17], "Evaluate")
-        self.assertEqual(crashInfo.backtrace[18], "JS::Evaluate")
-        self.assertEqual(crashInfo.backtrace[19], "EvalInContext")
-        self.assertEqual(crashInfo.backtrace[20], "??")
-        self.assertEqual(crashInfo.backtrace[21], "Print")
-        self.assertEqual(crashInfo.backtrace[22], "??")
-        self.assertEqual(crashInfo.backtrace[23], "??")
-        self.assertEqual(crashInfo.backtrace[24], "??")
-        self.assertEqual(crashInfo.backtrace[25], "??")
-        self.assertEqual(crashInfo.backtrace[26], "??")
-        self.assertEqual(crashInfo.backtrace[27], "??")
-        self.assertEqual(crashInfo.backtrace[28], "js::Activation::Activation")
-        self.assertEqual(crashInfo.backtrace[29], "EnterIon")
-        self.assertEqual(crashInfo.backtrace[30], "js::jit::IonCannon")
-        self.assertEqual(crashInfo.backtrace[31], "js::RunScript")
-        self.assertEqual(crashInfo.backtrace[32], "js::Invoke")
-        self.assertEqual(crashInfo.backtrace[33], "js::Invoke")
-        self.assertEqual(crashInfo.backtrace[34], "js::jit::DoCallFallback")
-        self.assertEqual(crashInfo.backtrace[35], "??")
-        self.assertEqual(crashInfo.backtrace[36], "EnterBaseline")
-        self.assertEqual(crashInfo.backtrace[37], "js::jit::EnterBaselineMethod")
-        self.assertEqual(crashInfo.backtrace[38], "js::RunScript")
-        self.assertEqual(crashInfo.backtrace[39], "js::ExecuteKernel")
-        self.assertEqual(crashInfo.backtrace[40], "js::Execute")
-        self.assertEqual(crashInfo.backtrace[41], "ExecuteScript")
-        self.assertEqual(crashInfo.backtrace[42], "JS_ExecuteScript")
-        self.assertEqual(crashInfo.backtrace[43], "RunFile")
-        self.assertEqual(crashInfo.backtrace[44], "ProcessArgs")
-        self.assertEqual(crashInfo.backtrace[45], "Shell")
-        self.assertEqual(crashInfo.backtrace[46], "main")
-        self.assertEqual(crashInfo.backtrace[47], "__tmainCRTStartup")
-        self.assertEqual(crashInfo.backtrace[48], "mainCRTStartup")
-        self.assertEqual(crashInfo.backtrace[49], "BaseThreadInitThunk")
-        self.assertEqual(crashInfo.backtrace[50], "RtlInitializeExceptionChain")
-        self.assertEqual(crashInfo.backtrace[51], "RtlInitializeExceptionChain")
+        self.assertEqual(len(crashInfo.backtrace), 33)
```

Comment:
> But you don't need to check them all :) You can still leave this as it is of course.

I'll probably leave these as-is. I had bulk-copied the stack entries and "massaged" them till they fit the testing syntax, rather than manually inputting them one-by-one.